client-app/core/api/graphql/cart/links/index.ts

```diff
@@ -1,5 +1,4 @@
 import { from } from "@apollo/client/core";
-import { batchLink } from "@/core/api/graphql/cart/links/batch";
 import { broadcastLink } from "@/core/api/graphql/cart/links/broadcast";
 
-export const cartLink = from([broadcastLink, batchLink]);
+export const cartLink = from([broadcastLink]);
```

Comment:
batchLink supposed to decrease the amount of requests - are we sure that we want to remove that?

Reply:
Yes, because implementation marks as Dirty hack.